src/cli/i18n.ts

```diff
@@ -14,7 +14,7 @@ const keywords = [
   'then',
   'and',
   'but',
-]
+] as const
```

Comment:
What makes this needed?

Reply:
Later in the file, we iterate over these keywords and get the value of each from a gherkin dialect object:

```ts
const language = Gherkin.dialects()[isoCode]
const rows = _.map(keywords, keyword => {
  const words = _.map(language[keyword], s => `"${s}"`).join(', ')
  return [capitalCase(keyword), words]
})
```

For the TS compiler to be okay with that, we have to add the `as const` here to tell it that the array won't be mutated, making it type safe because all of these keywords are properties of a dialect with string values.

Reply:
But the array is already saved into a const variable. Would providing a type to that like
```typescript
const keywords: string[] = [ /* ... */ ]
```
work?

Reply:
That doesn't quite do it because although declaring with `const` means it can never be reassigned, the array contents can still be modified in place:

![image](https://user-images.githubusercontent.com/3192745/74069598-a824fc00-49f6-11ea-963f-3a3fc93d393a.png)

The `as const` thing types it as an immutable array:

![image](https://user-images.githubusercontent.com/3192745/74069692-e91d1080-49f6-11ea-93cb-7e3cf44f4f8f.png)

This means the compiler is satisfied that it's safe to use the elements as keys of a dialect.

Reply:
Hmm okay. Don't think that is something we really need and dislike the uncommon syntax but I'm okay leaving it.